.github/mergify.yml

```diff
@@ -191,6 +194,7 @@ pull_request_rules:
         - "status-success!=Code Checker AMD64 Ubuntu 18.04"
         - "status-success!=Build and test AMD64 Ubuntu 18.04" 
         - "status-success!=Code Checker MacOS 11"    
+        - "status-success=Code Checker CentOS 7"
```

Comment:
Should be !=?

Reply:
modified already, please try to refresh and check again.